packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx

```diff
@@ -185,6 +196,52 @@ export const VAutocomplete = genericComponent<new <
       } else if (e.key === 'ArrowUp') {
         listRef.value?.focus('prev')
       }
+
+      if (!props.multiple) return
+
+      if (['Backspace', 'Delete'].includes(e.key)) {
+        if (selectionIndex.value < 0) {
+          if (e.key === 'Backspace' && !search.value) {
+            selectionIndex.value = length - 1
+          }
+
+          return
+        }
+
+        const originalSelectionIndex = selectionIndex.value
```

Comment:
This probably isn't needed because Autocomplete can't add custom values however it's not going to hurt anything having it in.